### PR TITLE
Fix hash render end with colon

### PR DESCRIPTION
**Pull request by lunny**
Fix a hash render problem like `<hash>: xxxxx` which is usually used in release notes.

### CODE REVIEW

**Review by appleboy — APPROVED** [review]

**Review by silverwind — APPROVED** [review]

File: modules/markup/html.go [hunk 1]
```diff
@@ -49,7 +49,7 @@ var (
 	// hashCurrentPattern matches string that represents a commit SHA, e.g. d8a994ef243349f321568f9e36d5c3f444b99cae
 	// Although SHA1 hashes are 40 chars long, SHA256 are 64, the regex matches the hash from 7 to 64 chars in length
 	// so that abbreviated hash links can be used as well. This matches git and GitHub usability.
-	hashCurrentPattern = regexp.MustCompile(`(?:\s|^|\(|\[)([0-9a-f]{7,64})(?:\s|$|\)|\]|[.,](\s|$))`)
+	hashCurrentPattern = regexp.MustCompile(`(?:\s|^|\(|\[)([0-9a-f]{7,64})(?:\s|$|\)|\]|[.,:](\s|$))`)
```
Comment (delvh) [inline modules/markup/html.go:52]: By the way, why aren't we using `\b` here?
I think that would simplify things…

Comment (lunny) [reply modules/markup/html.go:52]: Looks like yes. Maybe I should change to `\b`?

Comment (silverwind) [reply modules/markup/html.go:52]: Try it and see if the existing test cases work.

Comment (lunny) [reply modules/markup/html.go:52]: Maybe it can be tried by another PR since this one is a quick patch.

Comment (silverwind) [reply modules/markup/html.go:52]: Ok

**Pull request merged**